packages/website/src/components/lib/markdown.ts

```diff
@@ -29,25 +29,51 @@ function createSummaryJson(
   return '';
 }
 
+export function genVersions(state: ConfigModel): string {
+  return [
+    '| package | version |',
+    '| -- | -- |',
+    `| \`@typescript-eslint/eslint-plugin\` | \`${process.env.TS_ESLINT_VERSION}\` |`,
+    `| \`@typescript-eslint/parser\` | \`${process.env.TS_ESLINT_VERSION}\` |`,
+    `| \`TypeScript\` | \`${state.ts}\` |`,
+    `| \`ESLint\` | \`${process.env.ESLINT_VERSION}\` |`,
+    `| \`node\` | \`web\` |`,
+  ].join('\n');
+}
+
 export function createMarkdown(state: ConfigModel): string {
   return [
-    '## Repro',
     `[Playground](${document.location.toString()})`,
     createSummary(state.code, 'Code', 'ts', 30),
     createSummaryJson(state.rules, 'rules', 'Eslint config'),
     createSummaryJson(state.tsConfig, 'compilerOptions', 'TypeScript config'),
-    '## Expected Result\n',
-    '## Actual Result\n',
-    '## Additional Info\n',
-    '## Versions',
-    `| package                            | version |
-| ---------------------------------- | ------- |
-| \`@typescript-eslint/eslint-plugin\` | \`${process.env.TS_ESLINT_VERSION}\` |
-| \`@typescript-eslint/parser\`        | \`${process.env.TS_ESLINT_VERSION}\` |
-| \`TypeScript\`                       | \`${state.ts}\` |
-| \`ESLint\`                           | \`${process.env.ESLINT_VERSION}\` |
-| \`Env\`                              | \`web\` |`,
+    genVersions(state),
   ]
     .filter(Boolean)
     .join('\n\n');
 }
+
+export function createMarkdownParams(state: ConfigModel): string {
+  const params = {
+    labels: 'bug,package:+eslint-plugin,triage',
```

Comment:
these are populated automatically based on the template - so we don't need to add them.